pkg/controllers/generic/export_syncer.go

```diff
@@ -142,6 +142,9 @@ func (f *exporter) SyncDown(ctx *synccontext.SyncContext, vObj client.Object) (c
 	pObj, err := f.patcher.ApplyPatches(ctx.Context, vObj, nil, f.config.Patches, f.config.ReversePatches, func(vObj client.Object) (client.Object, error) {
 		return f.TranslateMetadata(ctx.Context, vObj), nil
 	}, &virtualToHostNameResolver{namespace: vObj.GetNamespace(), targetNamespace: translate.Default.PhysicalNamespace(vObj.GetNamespace())})
+	if !kerrors.IsConflict(err) {
```

Comment:
Isn't the logic inverted here? Shouldn't this be `if kerrors.IsConflict(err) {`? Same goes for the other if statements below

Reply:
indeed. Copy and paste. Sec.

Reply:
fixed.